pkg/event/target/queuestore.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 const (
-	maxLimit = 10000 // Max store limit.
+	maxLimit = 1000000 // Max store limit.
```

Comment:
We should also change this in all Validate() functions of all targets

It should use the `maxLimit` global instead of hardcoding the value.